client/my-sites/hosting/style.scss

```diff
@@ -1,15 +1,12 @@
 @import "calypso/assets/stylesheets/shared/mixins/breakpoints";
 
-.preview-pane__content {
-	.hosting {
-		margin: 0;
-		padding: 0;
-		overflow-y: auto;
-		max-height: calc(100vh - 300px);
+.hosting {
+	max-width: 1040px;
```

Comment:
The width seems a bit narrow without nav redesign

![image](https://github.com/Automattic/wp-calypso/assets/13596067/13e0fbd3-1f60-4dbc-b6ef-c66c19671035)

Reply:
Not sure how you got to that state? This is the behavior in production:

<img width="783" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/dc8cf29a-46bb-4686-84fd-b9f7021ba603">

Reply:
Yes, otherwise it will be a bit complicated.

Do you still see any issue around this?

Reply:
I reverted the changes and took a different approach: https://github.com/Automattic/wp-calypso/pull/90320/commits/3307ada885dbf4fed1a4bb342c512224c706e6ef